client/app/scripts/superdesk-authoring/views/item-association.html

```diff
@@ -1,4 +1,4 @@
 <figure class="item-association" ng-class="{preview: related}">
     <img ng-if="related" ng-src="{{ related.renditions.viewImage.href }}" ng-click="edit(related)">
-    <figcaption contenteditable="true" ng-model="related.description" ng-change="onchange()"></figcaption>
+    <figcaption contenteditable="true" ng-model="related.description_text" ng-change="onchange()"></figcaption>
```

Comment:
I've changed this also on server, it is description_text in pa and in ninjs standard, in is published like that. there should be no `description` field